td.server/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.5.0",
   "private": true,
   "scripts": {
+    "predev:vue": "node td.vue/src/plugins/generate-cornucopia.js",
```

Comment:
I think there is no need to have this script, so it can be reverted
So no change to this file and also no change to `td.server/package-lock.json` which would have to have been updated if we were to use `npm --clean-install`